validation/src/sobol_estimator_variance/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.png
+__pycache__
+graphe_validation/
+graphe_validation_mean_distribution/
+
```

Comment:
perhaps a .gitignore in /validation would be best, to avoid having one per folder in the future

Reply:
Ok. I create a new .gitignore in the parent directory with that particular content. This makes sense because there are Python scripts here, e.g. ValidPareto.py